poisson_lib.py

```
@@ -72,6 +72,44 @@ def run_poisson_regression(
         return None, None, None, str(e), stats_metrics
 
 
+def run_negative_binomial_regression(
+    y: Union[pd.Series, np.ndarray],
```

Ruff (0.14.10)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)